hls4ml/templates/quartus/firmware/weights/b2.h

```diff
@@ -0,0 +1 @@
+hls_init_on_powerup static const bias_default_t b2[config2::n_out] = {0.016205917468717468,0.8594570113367865,0.8702152750686849,0.17781786564910607,0.892286699539321,-0.481921867493851,0.3944938477209121,0.8171092371058477,-0.8344882848947686,-0.38468736037425466,-0.5344749363317189,-0.059873800340414896,-0.6397887543009495,-0.14078283158879978,-0.7373485450427875,0.6293532137862843};
```

Comment:
What's this? Some left over testing files? (Same for b2_test.h, w2.h, w2_test.h)

Reply:
Removed